### PR TITLE
Initial version of productInfo parser

### PR DESCRIPTION
**Pull request by tommorris**

### CODE REVIEW

File: parser/parser.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    count_of_tail_hashes = len(filter(has_hash, string_input.split(";;;")[1:]))
+    split_input = string_input.split('#', 1)
+
+    # MODE A PARSING
+    if string_input.count(";;;") > 0 and count_of_tail_hashes > 0 and string_input.split(';;;')[0].count('#') >= 2:
+        return {"title": split_input[0],
+                "items": parse_nested_hashes(split_input[1])}
+    # MODE B PARSING
+    elif count_of_tail_hashes == 0:
+        if len(split_input) is 1:
```
Comment (norm) [inline parser/parser.py]: `==` please, it is an arithmetic comparison, not an object comparison

Comment (tommorris) [inline parser/parser.py]: Fixed (along with the same on line 63).

File: README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+                'title': u'Style'},
+               {'items': [{'items': [{'items': [{'body': u'70% brass, 30% other'}],
+                                      'title': u'70% brass, 30% other'}],
+                           'title': u'Composition'}],
+                'title': u'Care and composition'}]}
+
+Tests
+-----
+
+    nosetests
```
Comment (norm) [inline README.md]: Could be a little more helpful to newcomers :)

**Comment by norm** [comment]
If you could use `git rebase` to:
* squash the first two commits and all subsequent gitignore commits into one
* squash "remove excess nesting"
* squash "adding requirements.txt - nose for testing"
* squash "is to == for arithmetic comparisons"
* squash "improved README"

then we're good to merge. And shout if you don't know what I mean or why I'm asking for that :)

**Comment by tommorris** [comment]
@norm: rebase complete.

**Pull request merged**